distributed/worker_state_machine.py

```diff
@@ -1129,7 +1130,7 @@ class WorkerState:
     #: :meth:`BaseWorker.gather_dep`. Multiple small tasks that can be fetched from the
     #: same worker will be clustered in a single instruction as long as their combined
     #: size doesn't exceed this value.
-    transfer_message_target_bytes: int
+    transfer_message_bytes_limit: float
```

Comment:
Driveby: Renamed to match other attribute names (e.g., `transfer_incoming_bytes_limit`). cc @crusaderky